### PR TITLE
Slashing import AT

### PR DESCRIPTION
**Pull request by jframe**

### CODE REVIEW

**Review by usmansaleem — APPROVED** [review]
LGTM

File: acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/slashing/SlashingImportAcceptanceTest.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+    final Path keyConfigFile = testDirectory.resolve("keyfile.yaml");
+    metadataFileHelpers.createUnencryptedYamlFileAt(
+        keyConfigFile, keyPair.getSecretKey().toBytes().toHexString(), KeyType.BLS);
+
+    startSigner(builder.build());
+  }
+
+  @Test
+  void slashingDataIsExported(@TempDir Path testDirectory) throws URISyntaxException {
```
Comment (rain-on) [inline acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/slashing/SlashingImportAcceptanceTest.java:71]: is this the right function name? I.e. appears to be importing, not exporting.

File: acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/slashing/SlashingImportAcceptanceTest.java [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    builder.withMode("eth2");
+    builder.withSlashingEnabled(true);
+    builder.withSlashingProtectionDbUrl(signer.getSlashingDbUrl());
+    builder.withSlashingProtectionDbUsername("postgres");
+    builder.withSlashingProtectionDbPassword("postgres");
+    builder.withKeyStoreDirectory(testDirectory);
+    builder.withSlashingImportPath(importFile);
+    builder.withHttpPort(12345); // prevent wait for Ports file in AT
+
+    final Signer importSigner = new Signer(builder.build(), null);
```
Comment (rain-on) [inline acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/slashing/SlashingImportAcceptanceTest.java:87]: why not map the file into the InterchangeFormat struct we have in a test fixture - and compare db against that?

means we don't need to compare directly against hard-coded expected values in the json file.

File: acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/slashing/SlashingImportAcceptanceTest.java (hunk 1 above)
Comment (jframe) [reply acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/slashing/SlashingImportAcceptanceTest.java]: whoops. fixed

File: acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/slashing/SlashingImportAcceptanceTest.java (hunk 2 above)
Comment (jframe) [reply acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/slashing/SlashingImportAcceptanceTest.java:92]: done

**Pull request merged**